### DISCUSSION TITLE
想请问一下为什么只有Mac上的V2rayU可以成功联网（手机和windows都不行）

### DISCUSSION THREAD

**Discussion by CamaroW** [yanue/V2rayU#1449] [Q&A]
小白一名，自己在aws上弄了一个最简单的服务器，装了一个trojan和一些其他的东西，然后就一直在用自己的服务器代理了。

最近，手机上的小火箭和windows上的v2rayN都莫名其妙地翻不了墙了，小火箭还能“连通性测试”返回一个延迟，但就是上不了网。

另一个也用trojan的服务器就全部运行正常，唯独我现在在用的这个只能在mac上的v2rayU上用。

想问问v2rayU的代码里是有什么神奇之处吗，或者完全是我服务器的内部原因？

**Comment by yfan51** [comment]
![image](https://github.com/user-attachments/assets/f8cc4854-49a2-4070-a2e4-b8e05ef6d713)

**Reply by yfan51** [reply]
这个如何解决呢。打不开呀

**Reply by CamaroW** [reply]
去“设置 - 隐私与安全性 - （如果是新版本的macOS）往下翻有一个类似‘仍要打开’的选项“
